tests/lib/rules/no-invalid-html-attribute.js

```diff
@@ -373,6 +376,26 @@ ruleTester.run('no-invalid-html-attribute', rule, {
         },
       ],
     },
+    {
+      code: '<a rel="noreferrer        noopener"></a>',
+      output: '<a rel="noreferrer noopener"></a>',
+      errors: [
+        {
+          messageId: 'spaceDelimited',
+          data: { attributeName: 'rel' },
+        },
+      ],
+    },
+    {
+      code: '<a rel="noreferrer  noopener"></a>', // this string contains nbsp
```

Comment:
could we do this using a template literal and `'\xa0'`?

Reply:
It doesn't let me use template literal but it seems to work just fine with single quotes.
![eslint-error](https://user-images.githubusercontent.com/1742115/149669608-f8b2b6d6-dfc1-4a0b-b432-80081824b084.png)
![single-quotes](https://user-images.githubusercontent.com/1742115/149669609-61752c76-7648-44ba-a677-642263accaca.png)

Reply:
Changed in 4de38ef - should work with single quotes as well (test passing locally)

Reply:
oh sure, that works, i forgot this project doesn’t use babel